funcframework/framework.go

```diff
@@ -309,5 +310,5 @@ func writeHTTPErrorResponse(w http.ResponseWriter, statusCode int, status, msg s
 
 	w.Header().Set(functionStatusHeader, status)
 	w.WriteHeader(statusCode)
-	fmt.Fprint(w, msg)
+	fmt.Fprint(w, html.EscapeString(msg))
```

Comment:
Was the code scanning complaining about this one? If not, we probably want to leave it alone as there may be customers expecting their strings to be formatted a certain way and we're just changing it underneath them.